crates/swift-bridge-ir/src/bridged_type/bridgeable_result.rs

```diff
@@ -129,19 +154,31 @@ impl BuiltInResult {
         type_pos: TypePosition,
         types: &TypeDeclarations,
     ) -> String {
-        let convert_ok =
-            self.ok_ty
-                .convert_ffi_expression_to_swift_type("val.ok_or_err!", type_pos, types);
-        let convert_err =
-            self.err_ty
-                .convert_ffi_expression_to_swift_type("val.ok_or_err!", type_pos, types);
-
-        format!(
-            "try {{ let val = {expression}; if val.is_ok {{ return {ok} }} else {{ throw {err} }} }}()",
-            expression = expression,
-            ok = convert_ok,
-            err = convert_err
-        )
+        if self.ok_ty.is_null() {
```

Comment:
```rust
if let Some(zero_byte_encoding) = self.ok_ty.only_encoding() {
    let ok = zero_byte_encoding.swift;
    // ... you can use `return {ok}` instead of `return`
}
```

Same idea for the other `.is_null()` calls.